packages/gatsby/src/datastore/lmdb/lmdb-datastore.ts

```diff
@@ -42,12 +57,24 @@ function getDatabases(): ILmdbDatabases {
     databases = {
       nodes: rootDb.openDB({
         name: `nodes`,
+        // FIXME: sharedStructuresKey breaks tests - probably need some cleanup for it on DELETE_CACHE
```

Comment:
Do we have two stories in the backlog for the TODOs so that we don't forget it?

Reply:
Added in [ch33144]